test/aspnettests.js

```diff
@@ -115,10 +115,10 @@ describe('ASP.NET project file creation when kestrel command exists', function (
     before(function (done) {
         helpers.run(path.join( __dirname, '../generators/app'))
         .inTmpDir(function(dir) {
-            createTestProjectJson(dir, 'EXISTING_KESTREL_COMMAND'); })
+            createTestProjectJson(dir, 'EXISTING_WEB_COMMAND'); })
         .withLocalConfig(function() {
             return { "appInsightsOptIn": false, "runningTests": true }; })
-        .withPrompts({ projectType: 'aspnet', aspNetVersion: '1.0.0-beta8' })
+        .withPrompts({ projectType: 'aspnet', aspNetVersion: '1.0.0-rc1-final' })
```

Comment:
Ok, but tagged release (used by people) is rc1-final. That's not my fault:
https://www.youtube.com/watch?v=UvkJJGVq-b0&list=PL0M0zPgJ3HSftTAAHttA3JQU4vOjXFquF